beets/importer.py

```diff
@@ -620,8 +621,12 @@ def align_album_level_fields(self):
         """Make some album fields equal across `self.items`.
         """
         changes = {}
+        # Determine where to gather the info from for the RETAG action.
+        retag_asis = (self.choice_flag == action.RETAG and
+                      not self.items[0].artist and
+                      not self.items[0].mb_artistid)
```

Comment:
I don't quite follow why these fields are the right ones to check for this... is there a particular reason why you chose `artist` and `mb_artistid`?